packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -420,13 +420,17 @@ export interface SchemaCompatibilityStatus {
 	 * Whether the current view schema is sufficiently compatible with the stored schema to allow viewing tree data.
 	 * If false, {@link TreeView.root} will throw upon access.
 	 * @remarks
-	 * Currently, this field is true if `isEquivalent` is true as well as the following case:
+	 * If the view schema does not opt into supporting any additional cases, then `canView` is only true when `isEquivalent` is also true.
+	 * The view schema can however opt into supporting additional cases, and thus can also view documents with stored schema which would be equivalent, except for the following discrepancies:
```

Comment:
Nit: I'm not sure the "however" here is necessary.

Reply:
Ya, I wasn't sure about it being better than not having it, but it seems ok.